theme/templates/base.html

```diff
@@ -81,7 +81,11 @@
     <div class="row">
       <div class="col-md-8">
         <h3>Center for Open Neuroscience</h3>
+      <a href="https://ror.org/04tfhh831"><img alt="ROR logo" src="https://raw.githubusercontent.com/ror-community/ror-logos/main/ror-icon-rgb.svg" height="24" /> https://ror.org/04tfhh831</a>
```

Comment:
although -- could you please add a copy of that icon -- I do not want to rely on external resources... if anything - should have been from some commit or tag so if they move etc, doesn't disappear.  But a copy is better